lib/services/dapi/createDapi.js

```diff
@@ -23,7 +23,12 @@ async function createDapi(opts) {
   const authorizationToken = await getAwsEcrAuthorizationToken(options.getAwsOptions());
```

Comment:
You don't need to fetch token if you are not using it.

Reply:
changed